protocols/v2/codec-sv2/src/decoder.rs

```diff
@@ -1,3 +1,43 @@
+//! # Stratum V2 Codec Decoder
+//!
+//! This module provides functionality for decoding Stratum V2 messages, including support for
+//! the Noise protocol for secure communication.
+//!
+//! ## Features
```

Comment:
I usually feel the word "Feature" can be confusing, because in the context of Rust it usually means "feature flag" for `cargo`.

Reply:
Agree. What do you like instead? Optional Functionality, Configuration Options, Build Options, something else?

Reply:
Went with "Build Options".